geonode_mapstore_client/client/js/api/geonode/v2/index.js

```diff
@@ -243,6 +243,12 @@ export const getDocumentsByDocType = (docType = 'image', {
         }));
 };
 
+export const setFavouriteResource = (pk, method) => {
+    // eslint-disable-next-line dot-notation
+    return axios[method](parseDevHostname(`${endpoints[RESOURCES]}/${pk}/favorite`))
```

Comment:
Why do we need to pass the method?
I would prefer to have to separated function if they are needed

Reply:
to have one function, instead of 2 similar function, in this case change only the CRUD method

Reply:
If the aim is to use a single function I propose a different approach and do not expose the method.
Now we have here:

- `setFavouriteResource(pk, 'post')` // set favourite to true
- `setFavouriteResource(pk, 'delete')` // set favourite to false

My proposal is to make this more explicit:

- `setFavouriteResource(pk, true)` // set favourite to true
- `setFavouriteResource(pk, false)` // set favourite to false

Another proposal is to avoid as much as possible `eslint-disable`. We should try to follow the eslint rules:

```js
const setFavouriteResource = (pk, favourite) => {
    const request = favourite ? axios.post : axios.get;
    return request(parseDevHostname(`${endpoints[RESOURCES]}/${pk}/favorite`))
}
```